source/CIPlanetModel.cpp

```diff
@@ -30,16 +30,19 @@
  * @param unlocked The texture on the outside of a locked ring
  */
 void PlanetModel::setTextures(const std::shared_ptr<cugl::Texture>& core,
-                 const std::shared_ptr<cugl::Texture>& ring,
-                 const std::shared_ptr<cugl::Texture>& unlocked,
-                 const std::shared_ptr<cugl::Texture>& locked) {
+                              const std::shared_ptr<cugl::scene2::AnimationNode>& ring,
```

Comment:
Same comment here, why does did this need to be changed to an AnimationNode? Couldn't we just pass in the texture sheet?